util/jsonrpc-types/src/proposal_short_id.rs

```diff
@@ -26,7 +26,11 @@ impl ProposalShortId {
 
 impl From<packed::ProposalShortId> for ProposalShortId {
     fn from(core: packed::ProposalShortId) -> ProposalShortId {
```

Comment:
Should we inline this as recommended in https://github.com/nervosnetwork/godwoken/pull/629 ?

Reply:
```suggestion
    #[inline]
    fn from(core: packed::ProposalShortId) -> ProposalShortId {
```

Reply:
I don't understand this LTO feature, and if it proves to be a performance issue, it should be added in a separate PR.

Reply:
as far as i know, `inline` will provide more context for compiler/linker to know if  ` the length of the slice and just emit a simple load instruction most of the time`.
LTO likely will use `inline`, but if we clearly want to inline, inline it, only at cost of size(generate object size)